### PR TITLE
add jupytab-server to conda-forge

### PR DESCRIPTION
**Pull request by js-dieu**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by js-dieu** [comment]
ok for being maintainer

File: recipes/jupytab-server/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4ba7bb5b37374653febc0f279db970b80e964bda865cd1659977eecbc31cc7da
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install --no-deps . -vv"
```
Comment (marcelotrevisani) [inline recipes/jupytab-server/meta.yaml:14]:
```suggestion
  script: "{{ PYTHON }} -m pip install --no-deps . -vv"
  noarch: python
  entry_points:
    - jupytab=jupytab_server.jupytab:main
```

File: recipes/jupytab-server/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - jupyter_kernel_gateway
+
+test:
+  commands:
+    - jupytab --help  # [unix]
```
Comment (marcelotrevisani) [inline recipes/jupytab-server/meta.yaml:26]:
```suggestion
    - jupytab --help
    - pip check
  requires:
    - pip
  imports:
    - jupytab_server
```

**Comment by js-dieu** [comment]
@marcelotrevisani 
Thanks a lot for the input. I just adapted it slightly as this package is not meant for being imported.

**Pull request merged**